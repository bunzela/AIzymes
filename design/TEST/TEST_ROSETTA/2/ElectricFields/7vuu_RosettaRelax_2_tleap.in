source leaprc.protein.ff19SB 
    source leaprc.gaff
    loadamberprep   /home/bunzelh/AIzymes/design/TEST/Input/5TS.prepi
    loadamberparams /home/bunzelh/AIzymes/design/TEST/Input/5TS.frcmod
    mol = loadpdb /home/bunzelh/AIzymes/design/TEST/TEST_ROSETTA/2/ElectricFields/7vuu_RosettaRelax_2.pdb
    saveamberparm mol /home/bunzelh/AIzymes/design/TEST/TEST_ROSETTA/2/ElectricFields/7vuu_RosettaRelax_2.parm7 /home/bunzelh/AIzymes/design/TEST/TEST_ROSETTA/2/ElectricFields/7vuu_RosettaRelax_2.rst7
    quit
    