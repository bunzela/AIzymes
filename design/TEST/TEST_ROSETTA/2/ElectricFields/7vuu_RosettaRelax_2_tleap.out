-I: Adding /home/bunzelh/miniconda3/envs/py37/dat/leap/prep to search path.
-I: Adding /home/bunzelh/miniconda3/envs/py37/dat/leap/lib to search path.
-I: Adding /home/bunzelh/miniconda3/envs/py37/dat/leap/parm to search path.
-I: Adding /home/bunzelh/miniconda3/envs/py37/dat/leap/cmd to search path.
-f: Source /home/bunzelh/AIzymes/design/TEST/TEST_ROSETTA/2/ElectricFields/7vuu_RosettaRelax_2_tleap.in.

Welcome to LEaP!
(no leaprc in search path)
Sourcing: /home/bunzelh/AIzymes/design/TEST/TEST_ROSETTA/2/ElectricFields/7vuu_RosettaRelax_2_tleap.in
----- Source: /home/bunzelh/miniconda3/envs/py37/dat/leap/cmd/leaprc.protein.ff19SB
----- Source of /home/bunzelh/miniconda3/envs/py37/dat/leap/cmd/leaprc.protein.ff19SB done
Log file: ./leap.log
Loading parameters: /home/bunzelh/miniconda3/envs/py37/dat/leap/parm/parm19.dat
Reading title:
PARM99 + frcmod.ff99SB + frcmod.parmbsc0 + OL3 for RNA + ff19SB
Loading parameters: /home/bunzelh/miniconda3/envs/py37/dat/leap/parm/frcmod.ff19SB
Reading force field modification type file (frcmod)
Reading title:
ff19SB AA-specific backbone CMAPs for protein 07/25/2019
Loading library: /home/bunzelh/miniconda3/envs/py37/dat/leap/lib/amino19.lib
Loading library: /home/bunzelh/miniconda3/envs/py37/dat/leap/lib/aminoct12.lib
Loading library: /home/bunzelh/miniconda3/envs/py37/dat/leap/lib/aminont12.lib
----- Source: /home/bunzelh/miniconda3/envs/py37/dat/leap/cmd/leaprc.gaff
----- Source of /home/bunzelh/miniconda3/envs/py37/dat/leap/cmd/leaprc.gaff done
Log file: ./leap.log
Loading parameters: /home/bunzelh/miniconda3/envs/py37/dat/leap/parm/gaff.dat
Reading title:
AMBER General Force Field for organic molecules (Version 1.81, May 2017)
Loading Prep file: /home/bunzelh/AIzymes/design/TEST/Input/5TS.prepi
Loading parameters: /home/bunzelh/AIzymes/design/TEST/Input/5TS.frcmod
Reading force field modification type file (frcmod)
Reading title:
Remark line goes here
Loading PDB file: /home/bunzelh/AIzymes/design/TEST/TEST_ROSETTA/2/ElectricFields/7vuu_RosettaRelax_2.pdb
  total atoms in file: 523
  Leap added 479 missing atoms according to residue templates:
       479 H / lone pairs
Checking Unit.

/home/bunzelh/miniconda3/envs/py37/bin/teLeap: Warning!
The unperturbed charge of the unit (-11.000000) is not zero.

/home/bunzelh/miniconda3/envs/py37/bin/teLeap: Note.
Ignoring the warning from Unit Checking.

Building topology.
Building atom parameters.
Building bond parameters.
Building angle parameters.
Building proper torsion parameters.
Building improper torsion parameters.
old PREP-specified impropers:
 <5TS 64>:  C1   O3   N2   O4  
 <5TS 64>:  C6   C2   C1   N2  
 <5TS 64>:  C5   C1   C6   H01 
 <5TS 64>:  C3   C1   C2   H02 
 <5TS 64>:  C4   C2   C3   H03 
 <5TS 64>:  C3   C5   C4   O5  
 <5TS 64>:  C4   C6   C5   C9  
 <5TS 64>:  C5   H04  C9   N1  
 total 217 improper torsions applied
 8 improper torsions in old prep form
Building H-Bond parameters.
Incorporating Non-Bonded adjustments.
Not Marking per-residue atom chain types.
Marking per-residue atom chain types.
  (Residues lacking connect0/connect1 - 
   these don't have chain types marked:

	res	total affected

	CALA	1
	NLEU	1
  )
 (no restraints)
	Quit

Exiting LEaP: Errors = 0; Warnings = 1; Notes = 1.
